docs/tutorials/model_management.ipynb

```diff
@@ -0,0 +1,815 @@
+{
```

Comment:
<p>After that I would add a cell to explain the following:</p><p> </p><p>Note that <code>Models.select()</code> combine the different conditions with a AND operator. If one need to combine conditions with a OR operator they can use the <code>Models.selection_mask()</code> method to generate a boolean array that can be used for selection. For example:</p><p> </p><p>selection_mask = models_3fhl.selection_mask(name_substring="1748") | models_3fhl.selection_mask(name_substring="1731")</p><p>models = models_3fhl[selection_mask]</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3216/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>